otio_openassetio/operations/openassetio_media_linker.py

```diff
@@ -44,32 +44,45 @@ def link_media_reference(in_clip, media_linker_argument_map):
     # over time, however this function isn't, so we track a shared
     # session based on the arguments to the media linker.
     session_state = _sessionState(media_linker_argument_map)
-    manager = session_state.session.currentManager()
+    manager = session_state.manager
 
-    if not extract_one(manager.isEntityReference([mr.target_url])):
+    if not manager.isEntityReferenceString(mr.target_url):
         return
 
+    entity_reference = manager.createEntityReference(mr.target_url)
```

Comment:
[fix] It's better to use `createEntityReferenceIfValid` to avoid doubling up the check (as `createEntityReference` calls `isEntityReference`).

Reply:
[ddc907d](https://github.com/OpenAssetIO/otio-openassetio/pull/17/commits/ddc907ded94fcd21e4df6d4978e9a8a66d100146)